CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [3.1.0] (2024-08-14)
+
+### Enhancements
+- Add `:enabled` config value
```

Comment:
```suggestion
- Add `:enabled?` config value
```
boolean?

Reply:
I believe in this case it is unambiguous that `:enabled` is either `true` or `false`